spec/datadog/di/code_tracker_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "datadog/di/code_tracker"
+
+RSpec.describe Datadog::DI::CodeTracker do
+  let(:tracker) do
+    described_class.new
+  end
+
+  describe ".new" do
+    it "creates an instance" do
+      expect(tracker).to be_a(described_class)
+    end
+  end
```

Comment:
Sorry for asking, but what a point of this test? Technically we are testing here that Ruby is going to instantiate a class if we call new, but do we really challenge it?

Reply:
This test only exercises the constructor. If you object to it I can remove it.

Reply:
I think it makes sense to keep the test suite slim. I think we don't need to challenge ruby internals considering we have no additional logic to it

Reply:
Deleted.